pnpm-lock.yaml

```diff
@@ -401,6 +401,29 @@ importers:
       jest: 27.5.1
       next-auth: link:../next-auth
 
+  packages/adapter-xata:
+    specifiers:
+      '@next-auth/adapter-test': workspace:^0.0.0
+      '@next-auth/tsconfig': workspace:^0.0.0
+      '@prisma/client': ^3.10.0
```

Comment:
Lockfile might need to be regenerated, just run `pnpm install` in the root again after addressing https://github.com/nextauthjs/next-auth/pull/4911/files#r919517238